src/common/AdaptiveStream.cpp

```diff
@@ -131,9 +131,8 @@ bool adaptive::AdaptiveStream::DownloadImpl(const DownloadInfo& downloadInfo,
   std::map<std::string, std::string> headers = m_streamHeaders;
   headers.insert(downloadInfo.m_addHeaders.begin(), downloadInfo.m_addHeaders.end());
 
-  // Append stream parameters, only if not already provided
-  if (url.find('?') == std::string::npos)
```

Comment:
atm i dont remember exactly but im pretty sure that removing this will break other live streams
that already have the params along url because without this check will add and double params on the url

Reply:
Do you have any suggestion for the problem?